CHANGELOG.md

```diff
@@ -25,7 +27,6 @@ IMPROVEMENTS:
 
 BUG FIXES:
 
-  * Add `connectInject.overrideAuthMethodName` to allow setting the `-acl-auth-method flag` [[GH-278](https://github.com/hashicorp/consul-helm/pull/278)]
```

Comment:
This was already listed as an improvement (which I think it is rather than a bugfix)